soc/nxp/imxrt/soc.yml

```diff
@@ -38,6 +38,14 @@ family:
     - name: mimxrt685s
       cpuclusters:
       - name: cm33
+  - name: imxrt7xx
+    socs:
+    - name: mimxrt798s
+      cpuclusters:
+      - name: cm33_cpu0
+      - name: cm33_cpu1
+      - name: hifi1
+      - name: hifi4
```

Comment:
@nordicjm @tejlmand if these clusters not yet supported should they be in the soc.yml ?

Reply:
Would make sense to add them when support is added to keep things properly bisectable and minimal.

Reply:
So I need to remove hifi1 and hifi4 core provisionally, right? and add back once hifi core supported?

Reply:
Yes, doesn't make sense to be able to select a core in the build system that is impossible to use but will be listed as supported for a given board